package.nls.de.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "",
```

Comment:
And for Spanish, I'm thinking something like `Activar el tema de icono` but I'm leaving that to @robertohuertasm.